src/ui/popup/settings-page/devtools.tsx

```diff
@@ -79,6 +79,7 @@ export default function DevToolsGroup(props: ViewProps) {
                     class={{
                         'dev-tools-button': true,
                         'dev-tools-button--has-custom-fixes': hasCustomFixes,
+                        'external-link': true,
```

Comment:
It's rather a popup, not sure if this need a external-link

Reply:
The current arrow makes it look like another page will open up. It is someway or the other opening something new.

Reply:
Got it.